### ISSUE TITLE
Custom service doesn't respect nested service

### ISSUE THREAD

**Issue by NickBolles** [feathers-plus/generator-feathers-plus#108]
### Steps to reproduce
- Create a typescript app
- Create a custom service with typescript that is nested in a folder (such as `v1`)
- get the following compilation error in the `<servicename>class.ts` file
```
server/services/v1/auth-management/auth-management.class.ts:5:21 - error TS2307: Cannot find module '../../app.interface'.
5 import { App } from '../../app.interface';
```

### Expected behavior
Respect nested service folders. Not quite sure how this would work, But I would be willing to look at it in the near future (probably next week) if no one gets to it before then

**Comment by j2L4e** [comment]
@eddyystop there was an issue/pr about path mapping `@` to `/src`, but I can't seem to find it anymore. This would allow us to use `import { App } from '@/app.interface';` here.

**Comment by NickBolles** [comment]
@j2L4e this? https://github.com/feathers-plus/generator-feathers-plus/pull/29

**Comment by eddyystop** [comment]
Fixed with https://github.com/feathers-plus/generator-feathers-plus/issues/116

Published in cli+ v0.7.66 which runs generator v0.6.64.
